mne/simulation/tests/test_metrics.py

```diff
@@ -47,4 +47,4 @@ def test_metrics():
                   metric='foo')
 
 
-run_tests_if_main()
+run_tests_if_main()
```

Comment:
you added a weird character.

Reply:
it's just a removal of line break that is required at the end of any file